app/portainer/components/forms/git-form/git-form-auth-fieldset/git-form-auth-fieldset.html

```diff
@@ -1,6 +1,6 @@
 <div class="form-group">
   <div class="col-sm-12">
-    <por-switch-field ng-model="$ctrl.model.RepositoryAuthentication" label="Authentication" on-change="($ctrl.onChangeAuth)"></por-switch-field>
+    <por-switch-field ng-model="$ctrl.model.RepositoryAuthentication" label="Authentication" on-change="($ctrl.onChangeAuth)" datacy="component-gitAuthSwitch"></por-switch-field>
```

Comment:
should we also use `data-cy` instead of `datacy` here for consistency? And the corresponding variable should be `dataCy`.

Also apply to other `datacy`